src/Home.js

```diff
@@ -1,36 +1,36 @@
-import React, {Component} from "react";
-import './Home.css';
-class Home extends Component{
-    render(){
-        const arrow = "-------->"
-        return (
-          <div id="home" className="Home">
-            <div className="Home-welcome">Hello!</div>
-            <div className="Home-name">
-              <span>I'm </span>
-              <span className="Home-myName" style={{ color: "#A00" }}>
-                G
-              </span>
-              {"RISHMA".split("").map((ltr) => (
-                <div className="Home-myName">{ltr}</div>
-              ))}
-            </div>
-            <div className="Home-msg">Web Developer from India</div>
-            <div>
-              <a href="#projects">
-                <button className="Home-button"> View my Projects</button>
-              </a>
-              <a href="#about">
-                <button className="Home-button">About Me</button>
-              </a>
-            </div>
-            <div className="Home-scroll">
-              <div>SCROLL</div>
-              <div>{arrow}</div>
-            </div>
-          </div>
-        );
-    }
+import React, { Component } from "react";
+import "./Home.css";
+class Home extends Component {
+  render() {
+    const arrow = "-------->";
+    return (
+      <section id="home" className="Home">
+        <h1 className="Home-welcome">Hello!</h1>
+        <h3 className="Home-name">
+          <span>I'm </span>
+          <span className="Home-myName" style={{ color: "#A00" }}>
+            G
+          </span>
+          {"RISHMA".split("").map((ltr) => (
+            <div className="Home-myName">{ltr}</div>
+          ))}
+        </h3>
+        <p className="Home-msg">Web Developer from India</p>
+        <div>
+          <a href="#projects">
```

Comment:
question: It's still missing to improve the focus indicators for keyboard navigation, will you fix it in this PR or in another one? :)

Reply:
I have currently updated focus indicators for buttons and contact form. I haven't updated for icons yet as I am not sure of how to do it.